pkg/microservice/aslan/core/service/service/server_test.go

```diff
@@ -0,0 +1,39 @@
+package service
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/koderover/zadig/pkg/microservice/aslan/core/common/repository/models"
+)
+
+func TestUpdateWorkloads(t *testing.T) {
```

Comment:
use ginkgo/gomega to write your ut

Reply:
I use testify/assert ,simple for assert

Reply:
no, you should use ginkgo only, just for consistent